### PR TITLE
Ltmbr add seilbahnen bundeskonzession

### PR DESCRIPTION
**Pull request by marionb**
Test link:
https://mf-chsdi3.dev.bgdi.ch/ltmbr_add_seilbahnen_bundeskonzession/shorten/8c48b773fe

### CODE REVIEW

File: chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+</%def>
+
+<%def name="betreiber()">
+  <%
+  layer = 'ch.bav.seilbahnen-bundeskonzession.'
+  %>
+  <tr><td class="cell-left">${_(layer + 'betreiber_tuabkuerzung')}</td> <td>${c['attributes']['betreiber_tuabkuerzung'] or '-'}</td></tr>
+</%def>
+
+<%def name="layer(translation)">
```
Comment (b4l) [inline chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako:16]: In general it's a bad practice to name an attribute and a function the same in python as it gets overloaded.

File: chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<%def name="anlage()">
+  <%
+  layer = 'ch.bav.seilbahnen-bundeskonzession.'
```
Comment (b4l) [inline chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako:3]: Nice touch! Still, maybe just put it inline as it barely helps to increase readability or maintainability given there are only two instances or even only one further down.

**Review by b4l — COMMENTED** [review]
Looks good! Left some minor remarks. I like the translation mako-magic. I was wondering whether it would be possible to expose this and the layer name/bodid through the base mako.

File: chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako (hunk 1 above)
Comment (marionb) [reply chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako:16]: Thanks for the hint. Actually I did not want to use the variable any more, just the function.  I just forgot to change the code of this make. Good catch!

**Comment by marionb** [comment]
@b4l 
> I was wondering whether it would be possible to expose this and the layer name/bodid through the base mako.
I thought about that as well. It would be very handy (potentially and save some debugging time). I'll have a look but not in this PR.

**Review by faselm — APPROVED** [review]

**Comment by b4l** [comment]
@marionb did you redeploy and test your latest changes? Calling a function defined below does generally not work well in python, don't know about how it is handled in mako though.

**Pull request merged**